docs/install.rst

```diff
@@ -0,0 +1,58 @@
+Installing
+##############
+
+Installation guide.
+
+.. _install_requirements:
+
+Before starting
+===============
+
+We will require:
+
+  * git
+  * cmake
```

Comment:
This documentation is a place holder.
It has to be fleshed out with such information.